stacks/core/prod/env.d/knot-connector.env

```diff
@@ -1,5 +1,5 @@
 RABBITMQ_HOSTNAME=rabbitmq
 CLOUD_TYPE=KNOT_CLOUD
-CLOUD_SETTINGS={"protocol": "wss","hostname": "ws.knot.cloud","port":443,"uuid":"8285f12f-1edb-47d0-83ff-f89aefcf9a42","token":"d8ce5afd88f6a0039cb414a770d6a915e4716447"}
+CLOUD_SETTINGS=
```

Comment:
I think it's good documenting this configuration will be needed in development scenarios.